### PR TITLE
SSRC uae artifactory configurations

### PR DESCRIPTION
**Pull request by ssrcdevops**
Created a new workflow file "tiiuae-pixhawk-artifact-publish.yaml" by copying "tiiuae-pixhawk.yaml". 

Made necessary changes in the newly created workflow file for publishing artifacts to SSRC uae jfrog artifactory and pushing docker images to SSRC uae jfrog container registry.

### CODE REVIEW

File: .github/workflows/tiiuae-pixhawk-artifact-publish.yaml [hunk 1]
```diff
@@ -0,0 +1,277 @@
+name: tiiuae-pixhawk
```
Comment (jpaali) [inline .github/workflows/tiiuae-pixhawk-artifact-publish.yaml:1]: Please use some other name. This already exists.

File: .github/workflows/tiiuae-pixhawk-artifact-publish.yaml [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  pixhawk4:
```
Comment (jpaali) [inline .github/workflows/tiiuae-pixhawk-artifact-publish.yaml:12]: Do you also need binaries for pixhawk4?

**Review by jpaali — CHANGES_REQUESTED** [review]

File: .github/workflows/tiiuae-pixhawk-artifact-publish.yaml (hunk 1 above)
Comment (ssrcdevops) [reply .github/workflows/tiiuae-pixhawk-artifact-publish.yaml]: Name changed

File: .github/workflows/tiiuae-pixhawk-artifact-publish.yaml (hunk 2 above)
Comment (ssrcdevops) [reply .github/workflows/tiiuae-pixhawk-artifact-publish.yaml:12]: Yes

**Review by jpaali — APPROVED** [review]

**Comment by jlaitine** [comment]
I'd also rather see some real person doing commits than faceless "ssrcdevops" account in the future :) So please use your own github id to make things

**Pull request merged**